core/sds-aspect-model-serializer/src/main/java/io/openmanufacturing/sds/aspectmodel/serializer/PrettyPrinter.java

```diff
@@ -159,10 +162,33 @@ private Comparator<Map.Entry<String, String>> createPredefinedPrefixOrder() {
             .thenComparing( Map.Entry::getKey );
    }
 
+   private Properties loadProperties( final String filename ) {
+      final Properties properties = new Properties();
+      final InputStream propertiesResource = PrettyPrinter.class.getClassLoader().getResourceAsStream( filename );
+      try {
+         properties.load( propertiesResource );
+      } catch ( final IOException exception ) {
+         throw new RuntimeException( "Failed to load Properties: " + filename );
+      }
+      return properties;
+   }
+
+   private void showMilestoneBanner() {
+      // property file generated by properties-maven-plugin at build time
+      final Properties applicationProperties = loadProperties( "pom.properties" );
+      final String version = applicationProperties.get( "aspect-meta-model-version" ).toString();
+      if ( version.contains( "-M" ) ) {
+         writer.println( "# This model has been created by SDS SDK Version " + version + " and is not intended for productive usage." );
```

Comment:
The property loading logic does the right thing (it refers to the BAMM artifact's version) but the message is misleading, when it talks about "SDS SDK Version". Please replace this with: `"# This model was created using BAMM version " + version + " and is not intended for productive usage."`